src/FailureDetectorConfig.md

```diff
@@ -85,3 +88,65 @@ config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size",
 config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.min.std.dev.millis", "100");
 [...]
 ```
+
+### Ping Failure Detector
+
+This failure detector is based on `InetAddress.isReachable()`.
+When the JVM process has enough permissions to create RAW sockets, the implementation will choose to rely on ICMP Echo requests, whereas, if not enough permissions, it will fallback on attempting a TCP Echo on port 7. In the latter, both a successful connection or an explicit rejection, will be treated as `Host is Reachable`.
```

Comment:
.. port 7. In the latter case, both a successful connection or an explicit rejection will be treated as `Host is Reachable`.